.github/workflows/build-test.yaml

```diff
@@ -0,0 +1,92 @@
+name: Push to GCP Artifact Registry
```

Comment:
For your sake and debugging github runs make sure to change this before you merge :D

Reply:
Doh 🤦 addressed in https://github.com/Garvan-Data-Science-Platform/hpci/pull/3/commits/7f3edc6ed6d005bbfee589991f577e4673ac58f8